### PR TITLE
Update Multilang-protocol.md

### PR DESCRIPTION
**Pull request by MaurGi**
Adding list of supported languages and a link to the c# adapter we have published recently -

### CODE REVIEW

**Comment by arunmahadevan** [comment]
Looks good.

File: docs/Multilang-protocol.md [hunk 1]
```diff
@@ -7,6 +7,18 @@ This page explains the multilang protocol as of Storm 0.7.1. Versions prior to 0
 
 # Storm Multi-Language Protocol
 
+## Supported Lanugages
+
+Storm Multi-Language has implementation in the following languages:
+
+- [JavaScript](../storm-multilang/javascript)
```
Comment (HeartSaVioR) [inline docs/Multilang-protocol.md]: The docs will be compiled as html and placed to website, so docs should not refer outside of docs directory. You can describe maven artifacts instead, like
* Javascript: org.apache.storm / multilang-javascript
* Python: org.apache.storm / multilang-python
* Ruby: org.apache.storm / multilang-ruby

**Comment by MaurGi** [comment]
thanks for the feedback, modified and resubmitted -

File: docs/Multilang-protocol.md (hunk 1 above)
Comment (MaurGi) [reply docs/Multilang-protocol.md]: got it - fixed, thanks

**Comment by MaurGi** [comment]
@HeartSaVioR how do I squash an existing PR in GitHub? I don't see any option on the UI, do I make a new branch? thanks -

**Comment by HeartSaVioR** [comment]
@MaurGi OK never mind. I'll handle it during merging. Thanks for the contribution!

**Pull request merged**